docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -122,63 +122,188 @@ class Sidebar extends React.Component<any, any> {
     })
 */
 
+  getActiveCategoryIndex(at: String, sections: ShorthandValue<any>[]) {
```

Comment:
```js
const items = [
  {
    key: 'quickstart',
    title: { content: 'QuickStart', to: 'quick-start' },       
  },
  {
    key: 'faq',
    title: { content: 'FAQ', to: '/faq' },
    styles: treeItemStyles,
  },
]

_.findIndex(items, item => item.title.to === '/faq') // 1
```

You can use `_.findIndex()` from Lodash.

Reply:
Actually this function returns the parent of the item - a category. But I got your point, this Lodash method replaces my function getActiveTreeItem().

Reply:
And _.findIndex() is not working recursively so I cannot use it here